### PR TITLE
changing TB3 burger to waffle

### PR DESCRIPTION
**Pull request by SteveMacenski**
#1258 

Changing TB3 from burger to waffle, that we actually modeled and used

### CODE REVIEW

File: nav2_system_tests/worlds/turtlebot3_ros2_demo_obstacle.world [hunk 1]
```diff
@@ -83,7 +83,7 @@
           <pose>-0.032 0 0 0 0 0</pose>
           <geometry>
             <mesh>
-              <uri>model://turtlebot3_burger/meshes/burger_base.dae</uri>
+              <uri>model://turtlebot3_waffle/meshes/waffle_base.dae</uri>
```
Comment (crdelsey) [inline nav2_system_tests/worlds/turtlebot3_ros2_demo_obstacle.world]: These changes to the world files are only changing the visual elements associated with the model. All the sizes and weights used by the physics part of the engine are still the Burger model, aren't they?

I think the important part are the physical characteristics.

Comment (SteveMacenski) [reply nav2_system_tests/worlds/turtlebot3_ros2_demo_obstacle.world]: I suppose I misinterpreted the ticket. I thought all the params were for waffle but just had burger visuals. 

Will update

**Comment by SteveMacenski** [comment]
@crdelsey take a look again, I rectified them. 

FYI the physics of the TB3 is garbage, they didn't even bother to change the defaults on the mass or inertial properties.

**Comment by crdelsey** [comment]
> they didn't even bother to change the defaults on the mass or inertial properties.

Yeah. That's what I thought as well. The URDF file used as the Gazebo model in ROS 1 had values that looked like they had been calculated, but they weren't used in the SDF. I didn't have time to go through the math to determine if it mattered or not.

**Comment by SteveMacenski** [comment]
If you haven’t noticed anything wrong by now I think its fine. Just don’t run into stuff.

**Pull request merged**